src/main/scala/ai/privado/exporter/RepoConfigMetaDataExporter.scala

```diff
@@ -11,13 +11,19 @@ import ai.privado.model.Constants
 object RepoConfigMetaDataExporter {
 
   def getMetaData(cpg: Cpg, ruleCache: RuleCache) = {
-    val metaData   = mutable.LinkedHashMap[String, String]()
+    val metaData   = mutable.LinkedHashSet[mutable.HashMap[String, String]]()
     val configRule = ruleCache.getSystemConfigByKey(Constants.RepoPropertyConfig)
     try {
-      val propertySources = cpg.property.filter(p => p.name matches configRule).l
-      propertySources.foreach(p => {
-        metaData.put(p.name, p.value)
-      })
+      val propertySources = cpg.property
+        .filter(p => p.name matches configRule)
+        .l
+        .map(p => {
+          val configMap = mutable.HashMap[String, String]()
+          configMap.put(Constants.name, p.name)
+          configMap.put(Constants.value, p.value)
+          configMap.put(Constants.filePath, p.file.head.name)
+          metaData.addOne(configMap)
```

Comment:
Instead of created a variable `mutable metaData`, can we not return `mutable configMap` as part of the map and store the map result as `metaData`.

Usage of mutable object should be limited or not used.

Reply:
done.